### DISCUSSION TITLE
I have problem in production the map doesn'tshow

### DISCUSSION THREAD

**Discussion by jurip89** [uber/react-map-gl#2044] [Q&A]
I have deployed my app and the map doesn't show in production. the error is this one in the image.
Does someone have a solution, please?
<img width="2240" alt="Screenshot 2022-11-11 at 17 38 14 2" src="https://user-images.githubusercontent.com/85887553/201388965-72c857c8-f82a-4841-9b12-edeffda15df3.png">

**Comment by Pessimistress** [comment]
#1266
